tests/Feature/Laravel/Console/PanDeleteTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+beforeEach(function (): void {
+    $this->repository = mock(AnalyticsRepository::class)->makePartial();
```

Comment:
I like the test here, the way it's broken down. 
What is the performance gain in using a mock versus hitting the DB here? 
I would suggest hitting the DB and confirming the row is actually deleted.

Reply:
@ruchit288 can you assert the database is missing the ID after deleting in these tests instead of the mock? Another way would be, leave the command class test as it is, and have a separate test for the delete of method of the repository (assert database missing the ID deleted).  

You can check the merged branches, look at the feature test cases on how you can insert some data into the table. That would even prevent hard coding the `id` like `['id' => 1]`. You can use [insertGetId](https://laravel.com/docs/11.x/queries#auto-incrementing-ids), which would make the tests more stable and less prone to failure.